index.md

```diff
@@ -2,69 +2,127 @@
+    text: SkyWars
+    subtext: "Le Spawn"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
 
-  - image: img/Diapo/09.jpg
+  - image: img/Diapo/06.png
+    text: SkyWars
+    subtext: "Arène Nether"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
+
+  - image: img/Diapo/07.png
+    text: SkyWars
+    subtext: "Arène Ender"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
+
+  - image: img/Diapo/08.jpg
+    text: WitherMaze
+    subtext: "Le Spawn"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
+
+  - image: img/Diapo/09.png
+    text: Faction
+    subtext: "Le Spawn"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
+
+  - image: img/Diapo/10.png
+    text: Five Night's At Freddy's
+    subtext: "Le Spawn"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
+
+  - image: img/Diapo/11.png
+    text: Survival
+    subtext: "Le Spawn"
+    button:
+        text: "En savoir plus"
+        link: "https://www.youtube.com/watch?v=xG0Ju_pUHAA"
+        icon: "arrow-right-bold-hexagon-outline"
 
 games:
   - image: img/Vignettes/pvpsoup.png
     text: Pvp Soup
-    subtext: Le Pvp Soup est un mini-jeu où l'on choisit un kit pour devenir le roi de l'arène !
+    subtext: "Choisissez un kit pour devenir le roi de l'arène !"
     badge: Bataille Royale !
     link: "#!"
 
   - image: img/Vignettes/creatif.png
     text: Créatif
-    subtext: Le Créatif est un endroit où vous pouvez laisser libre cour à votre imagination et partager vos créations à vos amis !
-    badge: Imaginations !
+    subtext: Laissez libre cour à votre imagination et partager vos créations à vos amis !
     link: "#!"
 
   - image: img/Vignettes/skywars.png
     text: SkyWars
-    subtext: Le SkyWars est un mini-jeu où vous devez prendre le contrôle des îles aux alentours et être le survivant de cette bataille !
-    badge: Comme Robinson Crusoé mais dans le ciel !
+    subtext: Prenez le contrôle des îles et restez en vie  !
     link: "#!"
 
   - image: img/Vignettes/withermaze.jpg
     text: WitherMaze
-    subtext: Le WitherMaze est un mini-jeu où vous devez trouver le centre du labyrinthe, tuer le Wither et revenir à votre entrée pour déposer l'étoile du Nether !
-    badge: Inédit !
+    subtext: Trouvez l'entrée de la grotte du Wither dans un labyrinthe et terrassez-le !
+    badge: Seulement sur HappyBlocks !
```

Comment:
je sais mais je préfère seulement sur happyblocks car on a un public jeune, inédit est assimilé à un terme de la télévision,  il pourrait y avoir confusion en pensant inédit c'est que c'est nouveau pas forcément unique, avec "seulement sur happyblocks" on pense tout de suite que c'est unique c'est juste pour ça